src/bootstrap/test.rs

```diff
@@ -264,6 +264,7 @@ impl Step for Rls {
 
         let mut cargo = tool::prepare_tool_cargo(builder,
                                                  compiler,
+                                                 Mode::Rustc,
```

Comment:
This should be `Mode::ToolRustc` as we're building a tool

Reply:
Oops! let me fix this ASAP